keyboards/cmm_studio/saka68/readme.md

```diff
@@ -0,0 +1,12 @@
+# CMM.Studio Saka68
+
+CMM.Studio Saka68 Keyboard
```

Comment:
Do you have an image of the board you'd like to add here? We prefer an imgur link, with `l` appended to the filename to make it large thumbnail sized.

Reply:
added.

Reply:
> Do you have an image of the board you'd like to add here? We prefer an imgur link, with `l` appended to the filename to make it large thumbnail sized.

dont know why the link is not working. is it necessary?

Reply:
The image seems to have been removed.